commit b37e29d1459b70ab57b9ed5889232d787cae6da7
Author: bertin <macportsuser@macports.org>
Date:   Fri Oct 10 19:08:56 2025 +0200

    /opt/local/linux-ports/audio/audacity/files/patch-ffmpeg-functions.diff

diff --git a/modules/import-export/mod-ffmpeg/lib-ffmpeg-support/FFmpegFunctions.cpp b/modules/import-export/mod-ffmpeg/lib-ffmpeg-support/FFmpegFunctions.cpp
index 5f201e8fb..8d0100e29 100644
--- a/modules/import-export/mod-ffmpeg/lib-ffmpeg-support/FFmpegFunctions.cpp
+++ b/modules/import-export/mod-ffmpeg/lib-ffmpeg-support/FFmpegFunctions.cpp
@@ -329,12 +329,13 @@ std::vector<wxString> FFmpegFunctions::GetSearchPaths(bool fromUserPathOnly)
 
 #elif defined(__WXMAC__)
    paths.emplace_back(wxT("/Library/Application Support/audacity/libs"));
-   paths.emplace_back(wxT("/usr/local/lib/audacity"));
-   // x86_64 Homebrew
-   paths.emplace_back(wxT("/usr/local/lib"));
-   // ARM64 Homebrew
-   paths.emplace_back(wxT("/opt/homebrew/lib"));
 #endif
+   paths.emplace_back(wxT("@PREFIX@/lib/audacity"));
+   // MacPorts
+   paths.emplace_back(wxT("@PREFIX@/libexec/ffmpeg8/lib"));
+   paths.emplace_back(wxT("@PREFIX@/libexec/ffmpeg7/lib"));
+   paths.emplace_back(wxT("@PREFIX@/libexec/ffmpeg6/lib"));
+   paths.emplace_back(wxT("@PREFIX@/lib"));
 
    return paths;
 }
